python/mxnet/kvstore/kvstore.py

```diff
@@ -327,39 +344,37 @@ def pushpull(self, key, value, out=None, priority=0):
         key : str, int, or sequence of str or int
             Keys.
 
-        value : NDArray, RowSparseNDArray, list of NDArray or RowSparseNDArray,
-                or list of list of NDArray or RowSparseNDArray
+        value : NDArray, list of NDArray, or list of list of NDArray
             Values corresponding to the keys.
 
         out: NDArray or list of NDArray or list of list of NDArray
             Values corresponding to the keys.
 
         priority : int, optional
-            The priority of the pull operation.
-            Higher priority pull operations are likely to be executed before
-            other pull actions.
+            The priority of the operation.
```

Comment:
Does priority 0 mean the highest or the lowest? Please specify.

Reply:
the type is int. int can be positive or negative. So 0 is neither highest or lowest.

Reply:
So maybe we want to mention the priority is in ascending order? e.g. the larger the value the higher the priority?